packages/jupyterlab-chat-extension/src/index.ts

```diff
@@ -392,13 +392,16 @@ const chatCommands: JupyterFrontEndPlugin<void> = {
      *                Whether to open the chat in side panel or in main area.
      *  isPalette -   optional (default to false).
      *                Whether the command is in commands palette or not.
+     *  fromLauncher - optional (default to false).
+     *                Whether the command executed from the launcher.
      */
     commands.addCommand(CommandIDs.createChat, {
       label: args => (args.isPalette ? 'Create a new chat' : 'Chat'),
       caption: 'Create a chat',
       icon: args => (args.isPalette ? undefined : chatIcon),
       execute: async args => {
         const inSidePanel: boolean = (args.inSidePanel as boolean) ?? false;
+        const fromLauncher: boolean = (args.fromLauncher as boolean) ?? false;
```

Comment:
Do we need both ?
Looks like `inSidePanel` arg is used when creating a chat from the side panel, but is only transferred to the `CommandIDs.open` command currently. Maybe we could use it for the creation too ?

In addition, it would fix for free the chat commands ran from the `menu>new>chat` and from the command palette. Currently these 2 other ways of creating a chat still create it in default directory, we probably expect it to be in current directory ?

Reply:
@brichet but using `inSidePanel` to determine creation directory in all scenarios breaks some integration test https://github.com/jupyterlab/jupyter-chat/actions/runs/15847826256

Reply:
Indeed, the UI tests use the command palette for most of the chat creation

https://github.com/jupyterlab/jupyter-chat/blob/017da53ba7919505cee2255a5269f3736d3543ab/ui-tests/tests/test-utils.ts#L27-L29